bib/mybib.bib

```diff
@@ -56,6 +56,19 @@ @misc{pozyx-Positioning
 	journal={Pozyx NV}
 }
 
+@misc{pozyx-UWBObstacles,
+	title={Ultra-wideband and obstacles},
+	url={https://www.pozyx.io/technology/uwb-and-obstacles},
+	journal={Pozyx NV}
+}
+
+@misc{pozyx-AnchorHeights,
+	title={Where to place the anchors},
+	url={https://www.pozyx.io/technology/where-to-place-the-anchors},
+	journal={Pozyx NV}
+}
+
+
```

Comment:
```suggestion
@inproceedings{WaltemateThomas2016Tiol,
abstract = {<p><p>Latency between a user's movement and visual feedback is inevitable in every Virtual Reality application, as signal transmission and processing take time. Unfortunately, a high end-to-end latency impairs perception and motor performance. While it is possible to reduce feedback delay to tens of milliseconds, these delays will never completely vanish. Currently, there is a gap in literature regarding the impact of feedback delays on perception and motor performance as well as on their interplay in virtual environments employing full-body avatars. With the present study at hand, we address this gap by performing a systematic investigation of different levels of delay across a variety of perceptual and motor tasks during full-body action inside a Cave Automatic Virtual Environment. We presented participants with their virtual mirror image, which responded to their actions with feedback delays ranging from 45 to 350 ms. We measured the impact of these delays on motor performance, sense of agency, sense of body ownership and simultaneity perception by means of psychophysical procedures. Furthermore, we looked at interaction effects between these aspects to identify possible dependencies. The results show that motor performance and simultaneity perception are affected by latencies above 75 ms. Although sense of agency and body ownership only decline at a latency higher than 125 ms, and deteriorate for a latency greater than 300 ms, they do not break down completely even at the highest tested delay. Interestingly, participants perceptually infer the presence of delays more from their motor error in the task than from the actual level of delay. Whether or not participants notice a delay in a virtual environment might therefore depend on the motor task and their performance rather than on the actual delay.</p></p>},
author = {Waltemate, Thomas and Senna, Irene and Hülsmann, Felix and Rohde, Marieke and Kopp, Stefan and Ernst, Marc and Botsch, Mario},
booktitle = {Proceedings of the 22nd ACM Conference on virtual reality software and technology},
isbn = {9781450344913},
keywords = {Body Ownership ; Full-Body Motion Capture ; Latency ; Sense of Agency ; Simultaneity Perception ; Virtual Mirror},
language = {eng},
pages = {27-35},
publisher = {ACM},
series = {VRST '16},
title = {The impact of latency on perceptual judgments and motor performance in closed-loop interaction in virtual reality},
volume = {02-04-},
year = {2016},
}


```